python/src/uagents/models.py

```diff
@@ -1,19 +1,26 @@
 import hashlib
-import json
-from typing import Type, Union
+from typing import Any, Type, Union
 
-from pydantic import BaseModel
+from pydantic.v1 import BaseModel, Field
 
 
+# reverting back to pydantic.v1 BaseModel for backwards compatibility
 class Model(BaseModel):
+    @classmethod
+    def model_json_schema(cls) -> str:
+        return cls.schema_json()
+
+    def model_dump_json(self) -> str:
+        return self.json()
```

Comment:
Removed the `indent` and `sort_keys` to keep parity with v1
Example https://github.com/fetchai/uAgents/commit/9dc508d173dedc72ff3365692d4aafeb5f3e4de0#diff-93489236d31a4bf06237078577b1d223280256cf088366b24250fc36e62cf284L263